admin/tool/policy/db/install.xml

```diff
@@ -39,18 +39,19 @@
         <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true"/>
         <FIELD NAME="policyversionid" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false" COMMENT="ID of the policy document version"/>
         <FIELD NAME="userid" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false" COMMENT="ID of the user this acceptance is relevant to"/>
-        <FIELD NAME="status" TYPE="int" LENGTH="3" NOTNULL="false" SEQUENCE="false" COMMENT="Acceptance status: 0 - pending, 1 - accepted, -1 - rejected"/>
-        <FIELD NAME="language" TYPE="char" LENGTH="100" NOTNULL="true" SEQUENCE="false" COMMENT="Code of the language the user had when the policy document was displayed"/>
-        <FIELD NAME="usermodified" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="ID of the user who last modified the acceptance record"/>
+        <FIELD NAME="status" TYPE="int" LENGTH="1" NOTNULL="false" SEQUENCE="false" COMMENT="Acceptance status: 1 - accepted, 0 - not accepted"/>
+        <FIELD NAME="lang" TYPE="char" LENGTH="30" NOTNULL="true" SEQUENCE="false" COMMENT="Code of the language the user had when the policy document was displayed"/>
+        <FIELD NAME="usermodified" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false" COMMENT="ID of the user who last modified the acceptance record"/>
         <FIELD NAME="timecreated" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false" COMMENT="Timestamp of when the acceptance record was created"/>
-        <FIELD NAME="timemodified" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="Timestamp of when the acceptance record was last modified"/>
-        <FIELD NAME="token" TYPE="char" LENGTH="1333" NOTNULL="false" SEQUENCE="false" COMMENT="A string that can be used to authorize to acceptance of the policy (eg by parents)"/>
```

Comment:
So you are removing the token and the "pending" status. This was designed to allow policy documents be accepted on the user's behalf via email link, even without having an account in Moodle. Did I misunderstand something that we are dropping that feature completely? I did not understand it that we are going to force every parent to have a moodle account (which may have implications). I still assumed the admin will have a way to distribute the links.

Reply:
I meant to explain this but ran out of time yesterday and wanted to push my changes so we have less conflicts. 
I initially suggested token so that parents can agree to the child's policy by clicking the link in an email. However, later we decided that parents will be moodle users with the capability to accept on behalf in the user's context. We no longer store 'parentguardianemail' as the custom profile field of a child and we have nowhere to send the link with the token.

If we do want to add tokens later, I suggest that we have a separate table for the issued tokens, this way we can "expire" them, have multiple tokens for the same child (i.e. divorced parents), also store the email where the token was sent, who issued the token, comments, etc.

Reply:
I agree, good plan and better than the original design.